Rakefile

```diff
@@ -23,3 +23,15 @@ end
 task :build => [:dependencies, :submodules] do
   sh('bundle exec jekyll build --config _config.yml')
 end
+
+task :validate => [:build] do
+  # Explanation of arguments:
+  # --assume-extension  # Tells html-proofer that `.html` files can be accessed without the `.html` part in the url.
+  # --disable-external  # For speed. Ideally we'd check external links too, but ignoring for now.
+  # --empty-alt-ignore  # To avoid needing to fix lots upfront, we can migrate towards this later.
+  # --allow-hash-href   # Allow empty `#` links to mean "top of page". It's true that these can be errors, however we have far too many to really address this.
+  # --url-swap          # Adjust for Jekyll's baseurl. See https://github.com/gjtorikian/html-proofer/issues/618 for more.
+  # --url-ignore        # Allow mailto links without a target email, for our Share links. Works around https://github.com/gjtorikian/html-proofer/issues/552.
+  #                     # Allow old event links which aren't easily updated.
```

Comment:
Question: Should there be a documented argument here?

Reply:
Hrm, looks like this is a copy/pasta from the website PR. I updated the actually command line but not the docs. cddd33b fixes that.